koap-core/src/commonTest/kotlin/MessageTest.kt

```diff
@@ -25,11 +25,21 @@ import com.juul.koap.Message.Code.Response.ServiceUnavailable
 import com.juul.koap.Message.Code.Response.Unauthorized
 import com.juul.koap.Message.Code.Response.UnsupportedContentFormat
 import com.juul.koap.Message.Code.Response.Valid
+import com.juul.koap.Message.Option.Observe
+import com.juul.koap.Message.Option.Observe.Registration.Deregister
+import com.juul.koap.Message.Option.Observe.Registration.Register
 import kotlin.test.Test
 import kotlin.test.assertEquals
 
 class MessageTest {
 
+    @Test
+    fun optionToString() {
+        assertToString(Observe(Register), "Observe(value=0/Register)")
+        assertToString(Observe(Deregister), "Observe(value=1/Deregister)")
```

Comment:
I _think_ I prefer pipe `|` instead of the slash, in part because the slash could possibly be misread/confused for division (with it sitting right next to a number). 🤷 

```suggestion
        assertToString(Observe(Register), "Observe(value=0|Register)")
        assertToString(Observe(Deregister), "Observe(value=1|Deregister)")
```